src/views/SystemIntake/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import Header from 'components/Header';
+import './index.scss';
+
+type SystemProfileProps = {
+  match: any;
+};
+
+export const SystemIntake = ({ match }: SystemProfileProps) => {
+  return (
+    <div className="system-profile">
+      <Header activeNavListItem={match.params.profileId} name="INTAKE" />
+      <div className="grid-container">
+        <p className="system-profile__text">
+          The EASi System Intake process can guide you through all stages of
```

Comment:
Should we be pulling any copy out to its separate place rather than embedding it in views? I know in the Ruby world, there were gems that used config files for copy, so you could later add internationalization and whatnot.

Reply:
We could do this, but it was confirmed with CMS that internationalization is not intended to be supported. The only supported language is United States English.

Reply:
It might be worth our while to move the copy out into a separate place, but maybe not support internalization yet.

Reply:
Mikena, knowing this, should we pull it out? If there's not going to be any i18n (at least for the MVP)